workflow/usage.rst

```diff
@@ -138,7 +138,7 @@ like this:
 
     class BlogPost
     {
-        // This property is used by the marking store
```

Comment:
Is it the case with state machine as well?
State machine has only one place allowed at the same time